README.rst

```diff
@@ -2,13 +2,13 @@
   MDAnalysis Repository README
 ================================
 
-|numfocus| |build| |cov| [*]_
+|numfocus| |build| |cov| |mdanalysis| [*]_
 
 |docs| |devdocs| |usergroup| |developergroup| |anaconda| |mybinder|
 
-MDAnalysis_ is a Python library for the analysis of computer simulations of many-body systems at the molecular scale, spanning use cases from interactions of drugs with proteins to novel materials. It is widely used in the scientific community and is written by scientists for scientists. 
+MDAnalysis_ is an object-oriented Python library for the analysis of computer simulations of many-body systems at the molecular scale, spanning use cases from interactions of drugs with proteins to novel materials. It is widely used in the scientific community and is written by scientists for scientists. 
```

Comment:
We started de-emphasizing the "object oriented", so it's ok not to have it here.